addons/l10n_fr_invoice_addr/models/account_move.py

```diff
@@ -14,7 +14,7 @@ def _get_view(self, view_id=None, view_type='form', **options):
         company = self.env.company
         if view_type == 'form' and company.country_code in company._get_france_country_codes():
             shipping_field = arch.xpath("//field[@name='partner_shipping_id']")[0]
-            shipping_field.set('groups', '')
+            shipping_field.attrib.pop("groups")
```

Comment:
Would have been safer
```suggestion
            shipping_field.attrib.pop("groups", None)
```